### ISSUE TITLE
[MCH] suggestion if you're looking for things to do

### ISSUE THREAD

**Issue by idecibel** [Grammernatzi/XIVComboPlugin#86]
hello,
I was afraid you might be bored and was looking for work, so I decided to pay you another visit.

So, at least for me machinist is probably the coolest class of any MMO, robots, guns what is there not to like?
Originally I struggled with playing it because the heat blast spam was too taxing for me, but you delivered some enhancements which have taken care of all that.  My main problem now is that there are no buttons left and it's becoming borderline boring. 

With that in mind I'm trying to untick and use as little of the features as possible, but the godsend option to merge heat blast with the split shot combo ... requires the split shot combo option to be used, which makes sense, but I wonder if it would be possible to have an option where the latter is not a requirement? 

So TLDR would be an option where either:
a) Under hypercharge heat blast replaces split shot only.
b) or potentially heat blast replaces all the 1-2-3 shots individually, also possible.

This would enable keeping the 1-2-3 and at the same time have the good heat blast stuff.

**Comment by Grammernatzi** [comment]
Makes sense, I used to have a macro that did the first one before I used XIVCombo. So I can add that as a feature, I know people prefer having the combo be separated and instead more prefer the QoL features I make (because I really like to make a lot of them as of late), so I'll try to work on making things more compatible with that sort of play in the future.

**Comment by Grammernatzi** [comment]
Done, just went ahead and made it apply to all combo actions, and added an additional option that only changes your current combo action when overheated.

**Comment by idecibel** [comment]
works perfectly! great and thanks again!
